restaurant-recommender/full-rag-demo.ipynb

```diff
@@ -27,7 +27,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Install Packages"
+    "## Install Packages {#install-packages}"
```

Comment:
Why is this necessary? The headers are already linkable:
https://docs.tecton.ai/docs/tutorials/full_rag#install-packages

Reply:
They are not all linkable. Example Support Chat Bot, Simple Prompt, Features as Tools, Knowledge are not, for example

You are correct that some of these are already linkable, though. I will look into fixing the PR

Reply:
Okay I think it's because of the header number. `# Example Support Chat Bot` vs `## Install Packages`. Likely this is because Docusaurus expects the single # to indicate the title of the page. If we update all the headers here to have an extra `#`, I think we should be good to go 👍

Reply:
Good catch! Fixed